### PR TITLE
Add redis streams compatibility

### PR DESCRIPTION
**Pull request by darkgl0w**
Close https://github.com/fastify/fastify-redis/issues/31
- Use `ioredis@4.10.0` (allow to use Redis streams commands)
- Bump dependencies

#### Checklist

- [x] run `npm run test`
- [ ] tests are included
- [x] documentation is changed or added
- [x] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ function fastifyRedis (fastify, options, next) {
     fastify.redis[namespace] = client
   } else {
     if (fastify.redis) {
-      next(new Error('fastify-redis has already been registered'))
+      return next(new Error('fastify-redis has already been registered'))
```
Comment (Eomm) [inline index.js]: this is a fix.

In order to have a good (automatic) changelog in the GH release, could you move this line in another pr please?

Comment (darkgl0w) [reply index.js]: Done ^^.

File: package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "standard && tap test.js",
-    "redis": "docker run -p 6379:6379 --rm redis:3.0.7"
+    "redis": "docker run -p 6379:6379 --rm redis:latest"
```
Comment (Eomm) [inline package.json]: I would specify the version here.

@mcollina do you agree?

```suggestion
    "redis": "docker run -p 6379:6379 --rm redis:5"
```

Comment (darkgl0w) [reply package.json]: Done ^^

Comment (darkgl0w) [reply package.json]: Done ^^
